examples/tutorials/traitsui_4.0/items.py

```diff
@@ -158,7 +158,7 @@ class LeagueModelView(ModelView):
     got_hit = Button('Got a Hit')
 
     # The total number of hits
-    total_hits = Property(depends_on='model.teams.players.hits')
+    total_hits = Property(observe='model.teams.players.hits')
```

Comment:
I believe we need to insert `.items` on this lists here, eg. `model.teams.items.players.items.hits`

Reply:
same here - i undid this change.